README.md

```diff
@@ -1,2 +1,29 @@
 # aws-batch-jobs
```

Comment:
can we add some documentation about how to run these jobs? e.g. is there some infrastructure that has to be setup beforehand or does cloud automation handle setting up stuff?

Reply:
we will not run directly here. The repos only stores the source codes to build quay images. Cloud automation will handle the setup. Even in cloud-automation the setup is hidden from user perspective. User just need to run a command `gen3 bucket-manifest <input_bucket> <output_bucket`. No kube-setup-blabla at all